djautotask/sync.py

```diff
@@ -282,14 +282,31 @@ def _assign_field_data(self, instance, object_data):
         return instance
 
 
-class TicketSynchronizer(Synchronizer):
+class QueryConditionMixin:
+
+    def _get_query_conditions(self, query):
+        # Don't sync the 'Complete' status for tickets and tasks.
+        # Most if not all tickets/tasks end up here and stay here forever
+        # or until deleted.
+        # This would cause ticket/task syncs to take a very long time.
+        query.open_bracket('AND')
+        query.WHERE(
+            'Status',
+            query.NotEqual,
+            self.at_api_client.picklist['Ticket']['Status']['Complete']
```

Comment:
I can't remember if you told me already, did we confirm this is actually ok to do? Even though it is technically ticket? In theory it makes sense really.

Reply:
Atws doesn't make `self.at_api_client.picklist['Task']['Status']` available by this method. I guess because the statuses are the same. I think its ok to use this method here since the statuses are the same and we don't have any reason to think that might change soon. Still, we can access the picklist values directly from the Task if we need to, just not using this method.

Reply:
Great! That makes sense and seems like this is the intentional way then!